cmd/synthetic-monitoring-agent/main.go

```diff
@@ -176,6 +171,16 @@ func run(args []string, stdout io.Writer) error {
 		zerolog.SetGlobalLevel(zerolog.ErrorLevel)
 	}
 
+	baseCtx, err := tracing.Context(context.Background())
+	if err != nil {
+		zl.Err(err).Msg("could not configure tracer provider, disabling tracing functionality")
+	}
```

Comment:
This is what I mean elsewhere: I read this code and I go like "disabling? you aren't disabling anything! I can see you are using the context as if it was OK".

Reply:
I am using the context as if it is okay because it _is_ okay: It either has a working tracerprovider, or a noop tracerprovider, but there will be one. The log message is just telling a white lie because "using noop tracerprovider" is probably more confusing to users.